api/src/main/kotlin/com/mashup/dojo/dto/QuestionDto.kt

```diff
@@ -0,0 +1,23 @@
+package com.mashup.dojo.dto
+
+import com.mashup.dojo.domain.QuestionType
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotBlank
+import jakarta.validation.constraints.NotNull
+
+@Schema(description = "질문 등록 요청")
+data class QuestionCreateRequest(
+    @field:NotBlank
+    val content: String,
+    @field:NotNull
+    val type: QuestionType,
+    @field:NotBlank
+    @Schema(description = "질문 이미지")
+    val imageUrl: String,
```

Comment:
```suggestion
    @Schema(description = "질문 이미지 id")
    val imageId: String,
```
이미지 업로드 API를 별도로 두게 됨으로서 이미지 id를 주고받는게 좋을 것 같아요!! (질문지에 첨부된 이미지를 수정할 수도 있으니.. )